src/services/patient.js

```diff
@@ -53,6 +57,10 @@ export const UpdateMessageStatus = (hashKey) => {
   return putData(`/register-patient-intake/message-status/${hashKey}`);
 };
 
+export const UpdateAppointmentStatus = (appointmentId, patientId) => {
+  return putData(`/appointment/${appointmentId}/check-in/${patientId}`);
```

Comment:
NIT: if this api call only changes the status of the appointment, it's all right to use a `patch` request instead of `put`

Reply:
then entire task is yet to be completed so there might be changes needed to this in future. Will take care of this later if nothing changes